app/src/test/java/com/example/plantsapp/presentation/ui/plantcreation/PlantCreationValidatorTest.kt

```diff
@@ -83,7 +83,7 @@ class PlantCreationValidatorTest {
         val validatorResult = validator.validate(
             plantName = "Jackob",
             speciesName = "",
-            frequency = null
+            wateringFrequency = null
         )
```

Comment:
I think it is time to add some factory method for validator, because we skipped it last time, and now look - we need to change a lot of lines of code in test instead of only one

Reply:
fix tests